lib/ukintctlr/ukintctlr.c

```diff
@@ -44,13 +45,19 @@
 #define MAX_IRQ				UK_INTCTLR_MAX_IRQ
 #define MAX_HANDLERS_PER_IRQ		CONFIG_LIBUKINTCTLR_MAX_HANDLERS_PER_IRQ
 
+#define ALLOCABLE_IRQ_COUNT		UK_INTCTLR_ALLOCABLE_IRQ_COUNT
```

Comment:
Why not use `ALLOCATABLE` instead of `ALLOCABLE` everywhere? :thinking:

Reply:
That was from the original PR by @mschlumpp so I kept it as is. I speculate it's for keeping the definition shorter and not have to deal so much with the line limit?

Reply:
Where is allocatable used?

Reply:
I mean it's a 2 letter difference, thought we may as well use the correct word. But I have not strong feelings towards this, proceed however you want.

Reply:
Fun fact: https://en.wiktionary.org/wiki/allocable

Reply:
Damn, you caught my english vocabulary lacking :tired_face: :rofl: